src/Cooling/test_cooling.cpp

```diff
@@ -44,8 +44,7 @@ template <> struct HydroSystem_Traits<CoolingTest> {
 template <> struct Physics_Traits<CoolingTest> {
   static constexpr bool is_hydro_enabled = true;
   static constexpr bool is_radiation_enabled = false;
-  static constexpr bool is_mhd_enabled = false;
-  static constexpr bool is_primordial_chem_enabled = false;
+  static constexpr bool is_chemistry_enabled = false;
   static constexpr bool is_metalicity_enabled = false;
```

Comment:
Can you remove the `is_metalicity_enabled` variable? I think we want to approach this somewhat differently. The user can add metallicity fields by setting `numPassiveScalars` for now.

Reply:
Yes, that should be removed now. At the moment only is_hydro_enabled, is_radiation_enabled, is_chemistry_enabled, and numPassiveScalars are defined in Physics_Traits.